twisted/conch/scripts/ckeygen.py

```diff
@@ -113,6 +113,8 @@ def generateDSAkey(options):
 
 
 def printFingerprint(options):
+    if not options.has_key('format'):
```

Comment:
we have 
```
optFlags = [['fingerprint', 'l', 'Show fingerprint of key file.'],
```

so fingerprint is a flag.. it will never has a value.

you should make it an `optParameters` with a default value